GitMeFood/ContentView.swift

```diff
@@ -32,8 +32,10 @@ struct ContentView: View {
                     ForEach(foodItems, id: \.self) { food in
                         Label(food, systemImage: "fork.knife.circle")
                     }.onMove { indexSet, newIndex in
+                        #warning("TODO: Add logic for moving items")
                         print("moved item in \(indexSet) to \(newIndex)")
                     }.onDelete { indexSet in
+                        #warning("TODO: Add logic for deleting items")
```

Comment:
Nice job adding these handlers. I'd suggest adding handlers to the bottom section too.

Reply:
@orgcottontee - did this change work for you?

Reply:
Looks good!